### PR TITLE
Adding how to get started.

### PR DESCRIPTION
**Pull request by DexterInd**
Adding how to run this in detail with commands.

### CODE REVIEW

**Comment by DexterInd** [comment]
@CleoQc can you please comment on the changes?

File: jessie_update/README.md [hunk 1]
```diff
@@ -3,21 +3,21 @@
 This directory contains the scripts we used to upgrade Jessie to Raspbian for Robots.  This is simply here for development purposes and so that it might be found to be useful to others.
 
 # To Begin
-+From the home directory run `sudo wget https://raw.githubusercontent.com/DexterInd/Raspbian_For_Robots/master/jessie_update/regimen.sh`
++ From the home directory run `sudo wget https://raw.githubusercontent.com/DexterInd/Raspbian_For_Robots/master/jessie_update/regimen.sh`
 
-+From the home directory run `sudo sh regimen.sh`
++ From the home directory run `sudo sh regimen.sh`
```
Comment (CleoQc) [inline jessie_update/README.md]: No longer required. The libraries are being removed in cleanup anyway. With the use of PiClone, this step can be skipped

File: jessie_update/README.md [hunk 2]
```diff
@@ -3,21 +3,21 @@
...
 
-+From the home directory run `sudo wget https://raw.githubusercontent.com/DexterInd/Raspbian_For_Robots/master/jessie_update/jessie-heavy-upgrade.sh`
++ From the home directory run `sudo wget https://raw.githubusercontent.com/DexterInd/Raspbian_For_Robots/master/jessie_update/jessie-heavy-upgrade.sh`
 
-+From the home directory run `sudo sh jessie-heavy-upgrade.sh`
++ From the home directory run `sudo sh jessie-heavy-upgrade.sh`
```
Comment (CleoQc) [inline jessie_update/README.md]: I'd `sudo bash` this, for safety.

File: jessie_update/README.md [hunk 3]
```diff
@@ -3,21 +3,21 @@
...
 
-+From the home directory run `sudo wget https://raw.githubusercontent.com/DexterInd/Raspbian_For_Robots/master/update_backup.sh`
++ From the home directory run `sudo wget https://raw.githubusercontent.com/DexterInd/Raspbian_For_Robots/master/update_backup.sh`
 
-+From the home directory run `sudo sh update_backup.sh`
++ From the home directory run `sudo sh update_backup.sh`
 
-+Run update_master.sh `sudo sh /home/pi/di_update/update_master.sh`
++ Run update_master.sh `sudo sh /home/pi/di_update/update_master.sh`
```
Comment (CleoQc) [inline jessie_update/README.md]: same here. `sudo bash`

File: jessie_update/README.md [hunk 4]
```diff
@@ -3,21 +3,21 @@
...
++ From the home directory run `sudo wget https://raw.githubusercontent.com/DexterInd/Raspbian_For_Robots/master/jessie_update/jessie-heavy-upgrade.sh`
 
-+From the home directory run `sudo sh jessie-heavy-upgrade.sh`
++ From the home directory run `sudo sh jessie-heavy-upgrade.sh`
 
-+From the home directory run `sudo wget https://raw.githubusercontent.com/DexterInd/Raspbian_For_Robots/master/update_backup.sh`
++ From the home directory run `sudo wget https://raw.githubusercontent.com/DexterInd/Raspbian_For_Robots/master/update_backup.sh`
 
-+From the home directory run `sudo sh update_backup.sh`
++ From the home directory run `sudo sh update_backup.sh`
```
Comment (CleoQc) [inline jessie_update/README.md]: same here `sudo bash`

File: jessie_update/README.md [hunk 5]
```diff
@@ -1,11 +1,36 @@
...
+
+## Manual Changes
+A few manual changes need to be made if you are starting from a fresh version of Raspbian:
+- VNC Password must be set manually.
+- Manual boot is required.
+- You MAY need to reconfigure /etc/hosts
 
 # What the files are:
-regimen.sh is used to remove some packages from a full Jessie so it can fit on 4 Gig. Some repartitioning must be done manually.
++ **regimen.sh** is used to remove some packages from a full Jessie so it can fit on 4 Gig. Some repartitioning must be done manually.
```
Comment (CleoQc) [inline jessie_update/README.md]: This is no longer needed with the advent of piclone

File: jessie_update/README.md [hunk 6]
```diff
@@ -1,11 +1,36 @@
...
+
++ Run update_master.sh `sudo sh /home/pi/di_update/update_master.sh`
+
++ Run VNC Server manually and enter in default passwords.  `vncserver :1`
+
++ From the Desktop, run the DI Update.
+
+## Manual Changes
+A few manual changes need to be made if you are starting from a fresh version of Raspbian:
+- VNC Password must be set manually.
```
Comment (CleoQc) [inline jessie_update/README.md]: run tightvncserver once and set the passwords
`tightvncserver`  no parameters
then run these lines:
https://github.com/DexterInd/Raspbian_For_Robots/blob/master/jessie_update/tightvncserver.sh#L22-L30

File: jessie_update/README.md (hunk 5 above)
Comment (DexterInd) [inline jessie_update/README.md]: If we're going to use piclone that should that be in these instructions as well.  @CleoQc can you submit a pr?

**Pull request merged**